app/services/scheduler.py

```
@@ -377,6 +374,27 @@ def _add_transfer_job_cleanup_job(self):
         except Exception:
             logger.exception("Error adding transfer job cleanup job")
 
+    def _add_p2p_manifest_sync_job(self):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_add_p2p_manifest_sync_job`

Add return type annotation: `None`

(ANN202)